Makefile

```diff
@@ -0,0 +1,43 @@
+ALL_PY_SRCS := setup.py \
+	$(shell find tourniquet -name '*.py') \
+	$(shell find tests -name '*.py')
+
+.PHONY: all
+all:
+	@echo "Run my targets individually!"
+
+.PHONY: dev
+dev:
+	test -d env || python3 -m venv env
+	. env/bin/activate && pip install -e .[dev]
+
+.PHONY: lint
+.ONESHELL:
+lint:
+	. env/bin/activate
+	black $(ALL_PY_SRCS)
+	isort $(ALL_PY_SRCS)
+	flake8 $(ALL_PY_SRCS)
+	git diff --exit-code
+
+.PHONY: typecheck
+.ONESHELL:
+typecheck:
+	. env/bin/activate
+	mypy tourniquet
```

Comment:
Minor thing that might not affect Tourniquet but maybe you want to add --ignore-missing-inputs if you ever add another dependency

Reply:
Yep, I'm tracking these in the `mypy.ini` file. It currently ignores `extractor`, since that's a black box from mypy's perspective.